cmd/nerdctl/run_windows.go

```diff
@@ -75,12 +77,30 @@ func setPlatformOptions(
 		opts = append(opts, oci.WithMemoryLimit(uint64(mem64)))
 	}
 
-	// TODO implement hyper-v isolation
 	isolation, err := cmd.Flags().GetString("isolation")
 	if err != nil {
 		return nil, err
 	}
 	switch isolation {
+	case "hyperv":
+		if memStr != "" {
```

Comment:
Are there defaults (are those in the runtime config?)

Reply:
Related, if no default was supplied I'm forgetting if the HCS v1 behavior was to expose the hosts cpu and memory size to the VM (if anyone wants to try this, otherwise I can report back tonight). With this change if you didn't supply any values the VM would have 2 cores and a similar low amount of ram. Could change this in a follow up though